cl/corpus_importer/utils.py

```diff
@@ -1,5 +1,5 @@
 from datetime import date
-from typing import Any, Optional
+from typing import Any, Iterator, List, Optional
```

Comment:
do we need List, I took a quick look thru the PEPs but didnt find the change but I think we dont need titled List here.  list will do

Reply:
@quevon24 https://peps.python.org/pep-0604/#simplified-syntax